cli/util/text.go

```diff
@@ -2,6 +2,7 @@ package util
 
 import "encoding/json"
 
+// IsJson checks if the input string is a valid json structure
 func IsJson(s string) bool {
 	var js map[string]interface{}
 	return json.Unmarshal([]byte(s), &js) == nil
```

Comment:
ok 😅🧐

Reply:
Not sure what bothers you here 😉

Reply:
nothing, it's just a bit cryptic to gather the meaning of these two lines ;) But you've documented it, so it's fine